pyalgotrading/algobulls/connection.py

```diff
@@ -455,13 +563,33 @@ def get_pnl_report_table(self, strategy_code, trading_type, country):
             _df['exit_transaction_type'] = _df['exit_transaction_type'].apply(lambda _: 'BUY' if _ else 'SELL')
             _df["pnl_cumulative_absolute"] = _df["pnl_absolute"].cumsum(axis=0, skipna=True)
 
+            # add slippage here
+            if slippage_percent:
+                if 'exit_variety' not in _df.columns or 'entry_variety' not in _df.columns:
+                    _df['exit_variety'] = 'MARKET'
+                    _df['entry_variety'] = 'MARKET'
+                    print('WARNING: Column for Order Variety not found. Assuming all trades are Market Orders.')
+
+                _df[['entry_price', 'exit_price']] = _df.apply(
+                    lambda row: (slippage(row.entry_price, row.entry_variety, row.entry_transaction_type, slippage_percent), slippage(row.exit_price, row.exit_variety, row.exit_transaction_type, slippage_percent)), axis=1, result_type='expand')
+
+                _df['pnl_absolute'] = _df['exit_price'] - _df['entry_price']
+
+            # add brokerage
+            _df['brokerage'] = ((_df['entry_price'] * _df['entry_quantity']) + (_df['exit_price'] * _df['exit_quantity'])) * (broker_commission_percentage / 100)
```

Comment:
There should be a way to turn off brokerage computation. Don't see it right now.

Reply:
If brokerage is not given then it will give 0 brokerage.